### DISCUSSION TITLE
.config file missing "auth": { section

### DISCUSSION THREAD

**Discussion by cheesecakedelight** [UltimaHoarder/UltimaScraper#750] [Q&A] [answered]
Exactly what the title says, I have no idea why this is happening. I'm new to python and I have version 3.9.1
When I do the python start_ofd.py command it gives me this,
![image](https://user-images.githubusercontent.com/77958596/105678623-5bfaf200-5ebb-11eb-82ea-f1dcfed7d14d.png)
Which is missing the necessary things I need to plug in the login info. 
I tried downloading the old auth.json and copy and pasting the code to the .config but it gets overwritten when I start the start_ofd.py command.
I also tried using this code instead: python -i start_ofd.py and it downloads the .profile folder and its files but it doesnt seem to do anything.
Any help is welcome and the FAQ didn't list this. I know the changelog listed "Fixed Auth issues" but I'm not sure if I discovered a new one.

**Comment by M-rcus** [comment, marked as answer]
The auth section has been moved to the `.profiles` directory, as described under ["Running the app locally"](https://github.com/DIGITALCRIMINAL/OnlyFans#running-the-app-locally)

**Reply by 3otgjrq** [reply]
I think the confusion might be coming up because the initial step asks you to mess with the config.json (which the old version did) but the new version generates the auth.json on the next step after hitting enter on the step where the program waits for you to modify the config.json. Confused me for a bit until it clicked and I tried that.
